### DISCUSSION TITLE
Regex problem

### DISCUSSION THREAD

**Discussion by sailingwhale** [DigitalRuby/IPBan#308] [Q&A]
I will set username blocklist rule with wildcards, something like this:
^(.*@domain.ext)$
but this not work :(

**Comment by jjxtra** [comment]
I'll create a unit test to verify user names are caught with regex and post back

**Reply by sailingwhale** [reply]
Any news?

**Reply by jjxtra** [reply]
Is TruncateUserNameChars config app setting set to @? If so, that may be why it's not matching.

**Reply by jjxtra** [reply]
Resolved in refreshed 2.0.1 update: https://github.com/DigitalRuby/IPBan/releases/tag/2.0.1

**Comment by sailingwhale** [comment]
no, its like on screenshot

**Reply by sailingwhale** [reply]
![Capto_Capture 2024-11-03_08-55-51_PM](https://github.com/user-attachments/assets/03a774a7-f5f9-4b11-89e0-153e0fdc6a02)

**Comment by sailingwhale** [comment]
I work with last DigitalRuby.IPBanProWebAdmin v.3.4.0 
API with 4 machines.

**Reply by jjxtra** [reply]
Have refreshed that version, too. You'll need to update each client. Tools menu -> update clients.
